org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -610,6 +618,30 @@ SsoApplicationManager:
         ]
       }
 
+SsoLlmDeveloper:
```

Comment:
This defines the permission set for LLM developers associated with the generic LLM developers group.